virtcontainers/sandbox.go

```diff
@@ -1049,14 +1053,6 @@ func (s *Sandbox) addContainer(c *Container) error {
 	}
 	s.containers[c.id] = c
 
-	ann := c.GetAnnotations()
-	if ann[annotations.ContainerTypeKey] == string(PodSandbox) {
```

Comment:
Note: We move the cgroup sandbox assigment from sandbox to cgroup to allow the container.newCgroups() can move the shim to the right cgroup of the pod if needed.